.github/workflows/rust.yml

```diff
@@ -84,9 +82,9 @@ jobs:
         components: rust-src
     - name: Install deps
       run: |
-        Start-BitsTransfer -Source https://ftp.mozilla.org/pub/mozilla/libraries/win32/MozillaBuildSetup-3.4.exe -Destination ./MozillaBuildSetup.exe
-        .\MozillaBuildSetup.exe /S | Out-Null
         echo 'C:\Program Files\Microsoft Visual Studio\2022\Enterprise\Tools\LLVM\bin' | Out-File -FilePath $env:GITHUB_PATH -Encoding utf8 -Append
+        curl -SL "https://github.com/servo/servo-build-deps/releases/download/msvc-deps/moztools-3.2.zip" --create-dirs -o target/dependencies/moztools.zip
+        cd target/dependencies && unzip -qo moztools.zip -d . && mkdir moztools && mv moztools-3.2 moztools/3.2
```

Comment:
I know this is a bit of a small thing, but I think we should stick to the environment variable here so that we are testing the method that most people will be using on the CI.

Reply:
You are right, this one will already be tested in servo.